content/documentation/pages/6-recipes/7-scaling/0-index.md

```diff
@@ -0,0 +1,29 @@
+---
+path: 'recipes/scaling/'
+title: 'Scaling'
+description: 'Using Prometheus and Data Flow Scale API for autoscaling pipelines'
+summary: true
+---
+
+# Scale-Out Data Pipelines
+
+[Scalability](https://en.wikipedia.org/wiki/Scalability) is the ability of a system to cope with increased load by adding resources. The [vertical](https://en.wikipedia.org/wiki/Scalability#Vertical) scaling (up/down) adds resources such as CPUs, memory and storage to the `existing nodes`, while the [horizontal](https://en.wikipedia.org/wiki/Scalability#Horizontal) scaling (out/in) adds `additional nodes` to the system.
+Following recipes demonstrate how to use `horizontal scaling` (e.g. `scaling out` or `shared nothing`) for streaming data pipelines.
```

Comment:
It won't be a recipe if we move it under features.

Reply:
Again, we don't need to highlight -> (e.g. `scaling out` or `shared nothing`) with backticks